README.md

```diff
@@ -1,70 +1,201 @@
+specifying one or more YARA rule. Your rules must have `meta` section with a 
+`score = [1-10]` tag to appropriately score matching binaries.  This directory is 
+configurable in your configuration file. C-style comments are supported.
 
-1. Stop the service: `service cb-yara-connector stop`
-2. Remove the database file: `rm /usr/share/cb/integrations/yara/db/sqlite.db`
-3. Remove the feed from your Cb server's Threat Intelligence page
-4. Restart the service: `service cb-yara-connector start`
+#### Sample YARA Rule File
+```
+// Sample rule to match binaries over 100kb in size
+
+rule matchover100kb {
+	meta:
+		score = 10
+	condition:
+		filesize > 100KB
+}
+```
 
-## Building yara-python with crypto
+## Controlling the YARA Agent 
+
+#### CentOS / Red Hat 6
+
+| Action | Command |
+| ------ | ------- |
+| Start the service | `service cb-yara-connector start` |
+| Stop the service | `service cb-yara-connector stop` |
+| Display service status | `service cb-yara-connector status` | 
+
+#### CentOS / Red Hat 7
+
+| Action | Command |
+| ------ | ------- |
+| Start the service | `systemctl start cb-yara-connector` |
+| Stop the service | `systemctl stop cb-yara-connector` |
+| Display service status | `systemctl status -l cb-yara-connector` |
+| Displaying verbose logs | `journalctl -u cb-yara-connector` |
+
+# Development Notes	
+
+## Utility Script
+Included with this version is a feature for discretionary use by advanced users and
+should be used with caution.
+
+When `utility_interval` is defined with a value greater than 0, it represents the interval
+in minutes at which the YARA connector will pause its work and execute an external
+shell script.  A sample script, `vacuumscript.sh`  is provided within the `scripts` folder
+of the current YARA connector installation. After execution, the YARA connector continues with
+its work.
+
+> _**NOTE:** As a safety for this feature, if an interval is defined but no script is defined, nothing is done.
+> By default, no script is defined._
+
+```ini
+;
+; The use of the utility script is an ADVANCED FEATURE and should be used with caution!
+;
+; If "utility_interval" is greater than 0 it represents the interval in minutes after which the YARA connector will
+; pause to execute a shell script for general maintenance. This can present risks. Be careful what you allow the
+; script to do, and use this option at your own discretion.
+;
+utility_interval=-1
+utility_script=./scripts/vacuumscript.sh
+```
 
-This is only needed if you are building the connector from scratch.
+## YARA Agent Build Instructions 
+
+The dockerfile in the top-level of the repo contains a CentOS 7 environment for running, building, and testing 
+the connector. 
+
+The provided script `docker-build-rpm.sh` will use docker to build the project, and place the RPM(s) in `${PWD}/RPMS`. 
+
+
+##### Command-line Options
+```text
+usage: yaraconnector [-h] --config-file CONFIG_FILE [--log-file LOG_FILE]
```

Comment:
Has this been renamed to cb-yara-connector? Or is this just gone?

Reply:
I ran `docker-build-rpm.sh` and it seemed to work very smoothly, producing an el7 RPM. How are el6 RPMs created?

Reply:
The executable is named `yaraconnector`.  That is the updated command-line options.